webui/react/src/pages/Clusters.tsx

```diff
@@ -57,7 +59,10 @@ const Clusters: React.FC = () => {
         <TabPane key="overview" tab="Overview">
           <ClustersOverview />
         </TabPane>
-        <TabPane key="logs" tab="Cluster Logs">
+        <TabPane className={css.tab} key="historical-usage" tab="Historical Usage">
```

Comment:
question: Why is historical usage the only tab pane with this CSS applied?

Reply:
Because `Historical Usage` itself does not have padding, it was wrapped into `Page` at another component.